include/revng/ABI/Predefined.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/ABI/Definition.h"
+
+namespace abi::predefined {
+
+const Definition &get(model::ABI::Values ABI);
```

Comment:
One day we'll switch to use a string as ABI identifier. So we can have a registry and let downstream users introduce new ones.

Reply:
Yep, definitely. But I don't see why that would be a problem.

Reply:
Closing this for now, since it's definitely out of scope.